<html>

<head>
  <link rel="stylesheet" type="text/css" href="style.css" />
  <title>
    Zongming Ma :: Research
  </title>
</head>

<body>

<table class="main" cellspacing=0 cellpadding=0>

<td width=15% class="topleft" valign="top"></td> 
<td class="middle" valign = "top">&nbsp;&nbsp;</td>
<td width=100% class="bottomrighttag" valign="top">
<h3>Zongming Ma</h3>
</td>

<!-- </table> -->

<!-- <table class="main" cellspacing=0 cellpadding=0> -->

<tr>
  <td width=15% class="topleft" valign="top"></td> 

  <td class="middle" valign = "top">&nbsp;&nbsp;</td>

</tr>

<tr>
<td width=5% class="bottomleftout" valign="top">
<table width=100% cellspacing=3>
<td class="bottomleft" valign="top">
<h4> &nbsp; <a href = "index.html"> Home </a> </h4>
</td>
</table>

<table width=100% cellspacing=3>
<td class="bottomleft" valign="top">
<h4> &nbsp; <a href = "people.html"> People </a> </h4>
</td>
</table>


<table width=100% cellspacing=3>
<td class="bottomleft" valign="top">
<h4> &nbsp; <a href = "research.html"> Research </a> </h4>
</td>
</table>

<table width=100% cellspacing=3>
<td class="bottomleft" valign="top">
<h4> &nbsp; <a href = "software.html"> Software </a></h4>
</td>
</table>

<table width=100% cellspacing=3>
<td class="bottomleft" valign="top">
<h4> &nbsp; <a href = "teaching.html"> Teaching </a></h4>
</td>
</table>

<td class="middle" valign = "top">&nbsp;&nbsp;</td>

<td width=100% class="bottomright" valign="top">


<!----------------------------------------------------------------------->
<!-- Main page -->

<h3>Research</h3>
</br>


<!-- <ul>
<li>High-dimensional statistical inference</li>
<li>Nonparametric statistics</li>
<li>Network data analysis</li>
<li>Statistical methods in genomics and imaging</li>
<!-- <li>Statistical machine learning</li> -->
<!-- </ul> -->


<h4>Some Preprints</h4>
<ul>
	



<li>
Enninful, A., Zhang, Z., Klymyshyn, D., Zong, H., Bai, Z., Farzad, N., Su, G., Baysoy, A., Nam, J., Yang, M., Lu, Y., Zhang, N. R., Braubach, O., Xu, M. L.,  Ma, Z., and Fan, R. (2024)<br>
<a href = "https://www.biorxiv.org/content/10.1101/2024.11.07.622523">
Integration of imaging-based and sequencing-based spatial omics mapping on the same tissue section via DBiTplus.
</a>
<br>
biorxiv:10.1101/2024.11.07.622523.
<!-- <em>Nat Immunol</em>. -->
<!-- [<a href="https://github.com/sggao/CellSNAP">Python Package</a>] -->
<br>
[AE & ZZ are co-first authors, MX, ZM & RF are co-correspondence authors]
</li>
<br>


<li>
Li, Y., Zhang, Z., Enninful, A., Nam, J., Qin, X., Villazon, J., Fazard, N., Fung, A. A., Xu, M. L., Jang, H., Zhang, N. R., Fan, R., Ma, Z., and Shi, L. (2024)<br>
<a href = "https://www.biorxiv.org/content/10.1101/2024.11.07.622511">
All-optical multimodal mapping of single cell type-specific metabolic activities via REDCAT.</a>
<br>
biorxiv:10.1101/2024.11.07.622511.
<!-- <em>Nat Immunol</em>. -->
<!-- [<a href="https://github.com/sggao/CellSNAP">Python Package</a>] -->
<br>
[YL, ZZ & AE are co-first authors, RF, ZM & LS are co-correspondence authors]
</li>
<br>


<li>
Sagnik, N. and Ma, Z. (2024)<br>
<a href = "http://arxiv.org/abs/2407.19030">
Multimodal data integration and cross-modal querying via orchestrated approximate message passing.</a>
<br>
arXiv:2407.19030.
<br> 
</li>
<br>


<li>
Chen, S., Jiang, S., Ma, Z., Nolan, G. P., and Zhu, B. (2022)<br>
<a href = "http://arxiv.org/abs/2204.13858">
One-way matching of datasets with low rank signals.</a>
<br>
arXiv:2204.13858.
<br> 
</li>
<br>



<li>
Banerjee, D. and Ma, Z. (2017)<br>
<a href = "http://arxiv.org/abs/1705.05305">
Optimal hypothesis testing for stochastic block models with growing degrees.
</a>
<br>
arXiv:1705.05305.
<br>
</li>
<br>

<!-- <li>
Tan, X. L., Buja, A., and Ma, Z. (2015)<br>
<a href = "http://arxiv.org/abs/1511.06821">
Kernel additive principal components.
</a>
<br>
arXiv:1511.06821.
<br>
</li>
<br> -->



</ul>



<!--</ol>-->

<h4>Journal Publications</h4>
<!--<ul>-->

<ul>
	
<li>
Gui, Y., Ma, C., and Ma, Z. (2025)<br>
<a href = "http://arxiv.org/abs/2505.12473">
Multi-modal contrastive learning adapts to intrinsic dimensions of shared latent variables.</a>
<br>
<!-- arXiv:2505.12473. -->
<em>NeurIPS</em>.
<br> 
</li>
<br>

	
<li>
Zhu, B., Gao, S., Chen, S., Wang, Y., Yeung, J., Bai, Y., Huang, A. Y., Yeo, Y. Y., Liao, G., Mao, S., Jiang, Z., Rodig, S. J., Wong, K.-C., Shalek, A. K., Nolan, G. P., Jiang, S., and Ma, Z. (2025)<br>
<a href = "https://www.nature.com/articles/s41590-025-02163-1">
CellLENS enables cross-domain information fusion for enhanced cell population delineation in single-cell spatial omics data.</a>
<br>
<!-- biorxiv:10.1101/2024.05.12.593710. -->
<em>Nat Immunol</em>, Vol.26, pp.963-974.
[<a href="https://github.com/sggao/CellSNAP">Python Package</a>]
<br>
[BZ, SG & SC are co-first authors, GPN, SJ & ZM are co-correspondence authors]
</li>
<br>

	
<li>
Zhang, Z., Mathew, D., Lim, T., Mason, K., Martinez, C. M., Huang, S., Wherry, E. J., Susztak, K., Minn, A. J., Ma, Z., and Zhang, N. R. (2024)<br>
<a href = "https://www.nature.com/articles/s41587-024-02463-1">
Recovery of biological signals lost in single-cell batch integration with CellANOVA.
<!-- Signal recovery in single cell batch integration. -->
</a>
<br>
<!-- biorxiv:10.1101/2023.05.05.539614. -->
<em>Nat Biotechnol</em>.
[<a href="https://github.com/Janezjz/cellanova">Python Package</a>]
<br>
[ZM & NRZ are co-correspondence authors]
</li>
<br>


<li>
Chen, S., Zhu, B., Huang, S., Hickey, J., Lin, K., Snyder, M., Greenleaf, W., Nolan, G. P., Zhang, N. R., and Ma, Z. (2024)<br>
<a href = "https://www.nature.com/articles/s41587-023-01935-0">
<!-- <a href = "https://www.biorxiv.org/content/10.1101/2023.01.12.523851v2"> -->
Integration of spatial and single-cell data across modalities with weakly linked features.
</a>
<br>
<!-- biorxiv:10.1101/2023.01.12.523851 -->
<em>Nat Biotechnol</em>, Vol.42, pp.1096-1106.
[<a href="https://github.com/shuxiaoc/maxfuse">Python Package</a>]
[<a href="https://maxfuse.readthedocs.io">Documentation</a>]
<br>
[SC & BZ are co-first authors, GPN, NRZ & ZM are co-correspondence authors]
</li>
<br>

<li>
Hickey, J. W., et al. (2023)<br>
<a href = "https://www.nature.com/articles/s41586-023-05915-x">
Organization of the human intestine at single cell resolution.
</a>
<br>
<!-- arXiv:2104.02960. -->
<em>Nature</em>, Vol.619, pp.572-584.
<br> 
</li>
<br>



<li>
Ma, Z. and Nandy, S. (2023)<br>
<a href = "http://arxiv.org/abs/2104.02960">
Community detection with contextual multilayer networks.</a>
<br>
<!-- arXiv:2104.02960. -->
<em>IEEE Trans Info Theory</em>, Vol.46(5), pp.3203-3239.
<br> 
</li>
<br>

<li>
Gao, S. and Ma, Z. (2023)<br>
<a href = "http://arxiv.org/abs/2107.00371">
Sparse GCA and thresholded gradient descent.</a>
<br>
<!-- arXiv:2107.00371. -->
<em>J Mach Learn Res</em>, Vol.24(135), pp.1âˆ’61.
<br> 
</li>
<br>



<li>
Ma, Z. and Yang, F. (2023)<br>
<a href = "http://arxiv.org/abs/2102.03297">
Sample canonical correlation coefficients of high-dimensional random vectors with finite rank correlations.</a>
<br>
<!-- arXiv:2102.03297. -->
<em>Bernoulli</em>, Vol.29(3), 1905-1932.
<br> 
</li>
<br>


	
<li>
Zhu, B., Chen, S., Bai, Y., Chen, H., Liao, G., Mukherjee, N., Vazquez, G., McIlwain, D. R., Tzankov, A., Lee, I. T., Matter, M. S., Goltsev, Y., Ma, Z., Nolan, G. P., and Jiang, S. (2023)<br>
<a href = "https://www.nature.com/articles/s41592-022-01709-7">
Robust single-cell matching and multi-modal analysis using shared and distinct features.</a>
<br>
<!-- bioRxiv:10.1101/2021.12.03.471185.  -->
<em>Nat Methods</em>. 
<!-- https://doi.org/10.1038/s41592-022-01709-7 -->
<!-- <br> -->
[<a href="https://github.com/shuxiaoc/mario-py">Python Package</a>]
<br> 
[BZ & SC are co-first authors, ZM, GPN & SJ are co-correspondence authors]
</li>
<br>





<!-- <li>
Hickey, J. W., <em>et al.</em> (2022+)<br>
<a href = "">
High resolution single cell maps reveals distinct cell organization and function across different regions of the human intestine.
</a>
<br>
<em>Nature</em>, accepted.
</li>
<br> -->

<li>
Gao, F., Ma, Z. and Yuan, H. (2022)<br>
<a href = "https://jmlr.org/papers/v23/20-1036.html">
Community detection in sparse latent space models.</a>
<br>
<!-- arXiv:2008.01375. -->
<em>J Mach Learn Res</em>, Vol.23(322), pp.1-50.
<br> 
</li>
<br>

	
<li>
Chen, S., Liu, S. and Ma, Z. (2022)<br>
<a href = "http://arxiv.org/abs/2012.00933">
Global and individualized community detection in inhomogeneous multilayer networks.</a>
<br>
<!-- arXiv:2012.00933. -->
<em>Ann Statist</em>, Vol.50(5), pp.2664-2693.
<br> 
</li>
<br>



	
<li>
Banerjee, D. and Ma, Z. (2022)<br>
<a href = "https://doi.org/10.1214/21-AOS2150">
Optimal signal detection in some spiked random matrix models: Likelihood ratio tests and linear spectral statistics.
<!-- Asymptotic normality and analysis of variance of log-likelihood ratios in spiked random matrix models. -->
</a>
<br>
<!-- arXiv:1804.00567. -->
<em>Ann Statist</em>, Vol.50(4), pp.1910-1932.
<br>
</li>
<br>

<li>
Chen, X., Li, X., and Ma, Z. (2022)<br>
<a href = "https://jmlr.org/papers/v23/21-1148.html">
Nonconvex matrix completion with linearly parameterized factors.</a>
<br>
<em>J Mach Learn Res</em>, Vol.23(207), pp.1-35.
<br>
</li>
<br>

	
<li>
Gao, C. and Ma, Z. (2022)<br>
<a href = "http://arxiv.org/abs/1910.12797">
Testing equivalence of clustering.</a>
<br>
<!-- arXiv:1910.12797. -->
<em>Ann Statist</em>, Vol.50(1), pp.407-429.
<br> 
</li>
<br>

	
<li>
Ding, J., Ma, Z., Wu, Y. and Xu, J. (2021)<br>
<a href = "http://arxiv.org/abs/1811.07821">
Efficient random graph matching via degree profiles.</a>
<br>
<em>Probab Theory Relat Field</em>, Vol.179, pp.29-115.
<!-- arXiv:1811.07821. -->
<br> 
</li>
<br>

	
<li>
Ma, Z., Ma, Z. and Yuan, H. (2020)<br>
<a href = "./papers/Latent.pdf"> 
Universal latent space model fitting for large networks with edge covariates.
</a>
<br>
<em>J Mach Learn Res</em>, Vol.21(4), pp.1-67.
<br>
</li>
<br>


	
<li>
Ma, Z., Ma, Z. and Sun, T. (2020)<br>
<a href = "http://arxiv.org/abs/1403.1922v2"> 
Adaptive estimation in two-way sparse reduced-rank regression.
</a>
<br>
<!-- arXiv:1403.1922v2. -->
<!-- <a href="./software/SARRScode.zip">[R Code]</a> -->
<em>Stat Sinica</em>, Vol.30, pp.2179-2201.
<br>
</li>
<br>


<li>
Xia, C. H., Ma, Z., Cui, Z., Bzdok, D., Basset, D. S., Satterthwaite, T. D., Shinohara, R. T., Witten D. (2020)<br>
<a href="https://doi.org/10.1002/hbm.24982"> 
Multi-scale network regression for brain--phenotype associations.
</a>
<br>
<!-- arXiv:1403.1922v2. -->
<!-- <a href="./software/SARRScode.zip">[R Code]</a> -->
<em>Hum Brain Mapp</em>, Vol.41(10), pp.2553-2566.
<br>
</li>
<br>



	
<li>
Gao, C., Ma, Z., Zhang, A. Y., and Zhou, H. H. (2018)<br>
<a href = "http://arxiv.org/abs/1607.06993"> 
Community detection in degree-corrected block models.
</a>
<br>
<!-- arXiv:1607.06993. -->
<em>Ann Statist</em>, Vol.46(5), pp.2153â€“2185.
<br>
</li>
<br>

	

<li>
Xia, C. H., Ma, Z., Ciric, R., Gu, S., Betzel, R., Kaczkurkin, A., Calkins, M., Cook, P., de la Garza, A. G., Vandekar, S., Cui, Z., Moore, T., Roalf, D., Ruparel, K., Wolf, D., Davatzikos, C., Gur, R., Gur, R., Shinohara, R., Bassett, D., and Satterthwaite, T. (2018)<br>
<a href = "https://www.nature.com/articles/s41467-018-05317-y"> 
Linked dimensions of psychopathology and connectivity in functional brain networks.
</a>
<br>
<em>Nat Commun</em>, Vol.9, Article number:3003.
<br>
</li>
<br>

	

	


	
<li>
Gao, C., Ma, Z., and Zhou, H. H. (2017)<br>
<a href = "http://arxiv.org/abs/1409.8565"> 
<!-- An efficient and optimal method for sparse canonical correlation analysis. -->
Sparse CCA: Adaptive estimation and computational barriers.
</a>
<br>
<!-- arXiv:1409.8565. -->
<em>Ann Statist</em>, Vol.45(5), pp.2074â€“2101.
<a href="./software/SCCALab.zip">[Matlab Code]</a>
<br>
</li>
<br>
	
<li>
Kennedy, E. H., Ma, Z., McHugh, M. D., and Small, D. S. (2017)<br>
<a href = "http://arxiv.org/abs/1507.00747"> 
Nonparametric methods for doubly robust estimation of continuous treatment effects.
</a>
<br>
<!-- arXiv:1507.00747. -->
<em>J Roy Stat Soc B</em>, Vol.79(4), pp.1229-1245.
<br>
</li>
<br>

<li>
Gao, C., Ma, Z., Zhang, A. Y., and Zhou, H. H. (2017)<br>
<a href = "http://arxiv.org/abs/1505.03772"> 
Achieving optimal misclassification proportion in stochastic block models.
</a>
<br>
<!-- arXiv:1505.03772. -->
<em>J Mach Learn Res</em>, Vol.18(60), pp.1-45.
<br>
</li>
<br>
	
<li>
Gao, C., Lu, Y., Ma, Z., and Zhou, H. H. (2016)<br>
<a href = "http://arxiv.org/abs/1512.00150"> 
Optimal estimation and completion of matrices with
biclustering structures.
</a>
<br>
<!-- arXiv:1512.00150. -->
<em>J Mach Learn Res</em>, Vol.17(161), pp.1-29.
<br>
</li>
<br>
	
<li>
Cai, T. T., Li, X., and Ma, Z. (2016)<br>
<a href = "http://arxiv.org/abs/1506.03382"> 
Optimal rates of convergence for noisy sparse phase retrieval via thresholded Wirtinger flow.
</a>
<br>
<!-- arXiv:1506.03382. -->
<em>Ann Statist</em>, Vol.44(5), pp.2221-2251.
<br>
</li>
<br>

<li>
Yang, D., Ma, Z., and Buja, A. (2016)<br>
<a href = "http://arxiv.org/abs/1405.0338"> 
Rate optimal denoising of simultaneously sparse and low rank matrices.
</a>
<br>
<!-- arXiv:1405.0338. -->
<em>J Mach Learn Res</em>, Vol.17(92), pp.1-27.
<br>
</li>
<br>

<li>
Ma, Z. and Wu, Y. (2015)<br>
<a href = "http://arxiv.org/abs/1306.3609">
Volume ratio, sparsity, and minimaxity under unitarily invariant norms.
</a><br>
<!-- arXiv:1306.3609. -->
<em>IEEE Trans Info Theory</em>, Vol.61(12), pp.6939-6956.
<br>
</li>
<br>




<li>
Gao, C., Ma, Z., Ren, Z., and Zhou, H. H. (2015)<br>
<a href = "http://arxiv.org/abs/1405.1595"> 
Minimax estimation in sparse canonical correlation analysis.
</a>
<br>
<!-- arXiv:1405.1595. -->
<em>Ann Statist</em>, Vol.43(5), pp.2168-2197.
<br>
</li>
<br>

<li>
Ma, Z. and Wu, Y. (2015)<br>
<a href = "http://arxiv.org/abs/1309.5914"> 
Computational barriers in minimax submatrix detection.
</a>
<br>
<!-- arXiv:1309.5914. -->
<em>Ann Statist</em>, Vol.43(3), pp.1089-1116.
<br>
</li>
<br>

<li>
Cai, T. T., Ma, Z., and Wu, Y. (2015)<br>
<a href = "http://arxiv.org/abs/1305.3235">
Optimal estimation and rank detection for sparse spiked covariance matrices.
</a><br>
<!-- arXiv:1305.3235.  -->
<em>Probab Theory Relat Field</em>, Vol.161(3), pp.781-815.
<br>
</li>
<br>





<li>
  Yang, D., Ma, Z., and Buja, A. (2014) <br>
  <a href = "http://arxiv.org/abs/1112.2433">A sparse SVD method for
  high-dimensional data.</a> <br>
<!-- arXiv:1112.2433.  -->
<em>J Comput Graph Stat</em>, Vol.23(4), pp.923-942. <br>
</li>
<br>

<li>
Cai, T. T., Low, M., and Ma, Z. (2014)<br>
<a href = "http://amstat.tandfonline.com/doi/full/10.1080/01621459.2013.879260">	Adaptive confidence bands for nonparametric regression functions.</a><br>
<em>J Amer Statist Assoc</em>, Vol.109, Issue 507, pp.1054-1070. <br>
<a href =
 "./papers/Adaptive_CB.pdf">[local version]</a>
<a href =
 "./papers/Adaptive_CB_supp.pdf">
[supplement]</a>
</li>
<br>

<li>
Cai, T. T., Ma, Z., and Wu, Y. (2013)<br>
<a href = "http://arxiv.org/abs/1211.1309">Sparse PCA: Optimal rates and
  adaptive estimation.</a><br>
<!-- arXiv:1211.1309. --> 
<em>Ann Statist</em>, Vol.41, pp.3074-3110. <br>
</li>
<br>


<LI>
  Ma, Z. (2013) <br>
  <a href = "http://projecteuclid.org/euclid.aos/1368018173">
	Sparse principal component analysis and iterative thresholding.</a> <br>
<!-- arXiv:1112.2432. --> 
<em>Ann Statist</em>, Vol.41, pp.772-801. <br>
<a href = "./papers/spcait.pdf">
	[complete version with supplement]</a>
<a href="./software/SPCALab/SPCALab.zip">[Matlab Code]</a> <br> 
</LI>
<br>

<LI>
  Cai, T. T. and Ma, Z. (2013) <br>
  <a href="http://arxiv.org/abs/1205.4219">Optimal hypothesis testing for high-dimensional covariance matrices.</a> <br>
 <em>Bernoulli</em>, Vol.19, pp.2359-2388. 
</LI>
<br>

<LI>
  Johnstone, I. M. and Ma, Z. (2012)<br>
  <a href="http://arxiv.org/abs/1110.0108">Fast approach to the Tracy-Widom law at the edge of GOE and GUE.</a><br>
  <em>Ann Appl Probab</em>, Vol.22, pp.1962-1988. 
  <a href="./papers/GOErate/GOErate.zip">[Matlab Code]</a> <br> 
</LI>
<br>

<LI>
  Ma, Z. (2012)<br>
  <a href="http://arxiv.org/abs/1203.0839">Accuracy of the Tracy-Widom limits for the extreme eigenvalues in white
  Wishart matrices.</a> <br>
  <em>Bernoulli</em>, Vol.18, pp.322-359. 
  <!-- [An <a href="http://arxiv.org/abs/0810.1329">earlier version</a> is -->
  <!-- available on arXiv</a>] --> 
  <a href="./papers/LOErate/twacc_supp.pdf">[supplement]</a> 
  <a href="http://cran.r-project.org/web/packages/RMTstat/index.html">[R
    package]</a>
  <br>
</LI>
<br>

<li> 
  Zhao, H., Ma, Z., Tibshirani, R., Higgins, J. P. T., Ljungberg, B., and
  Brooks, J. D. (2009)<br>
  Alteration of gene expression signatures of cortical differentiation and
  wound response in lethal clear cell renal cell carcinomas. <br>
  <em>PLoS ONE</em> 4(6): e6039.
</li>
<br>


<LI>
  Ma, Z., Xie, X., and Geng, Z. (2008)<br>
  <a href="http://jmlr.csail.mit.edu/papers/volume9/ma08a/ma08a.pdf">Structural learning of chain graphs via decomposition.</a> <br>
  <em>J Mach Learn Res</em>, Vol.9, pp.2847-2880.
  <a href="http://cran.r-project.org/web/packages/lcd/index.html">[R package]</a><br>
</LI>
<br>

<LI>
  Xie, X., Ma, Z., and Geng, Z. (2008)<br>
  <a href="http://www3.stat.sinica.edu.tw/statistica/J18N3/J18N319/J18N319.html">Some association measures and their collapsibility.</a><br>
  <em>Stat Sinica</em>, Vol.18, pp.1165-1183.
  <br>
</LI>
<br>

<LI>
  Ma, Z., Xie, X., and Geng, Z. (2006)<br>
  <a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1467-9868.2005.00536.x/abstract">Collapsibility of distribution dependence.</a><br> 
  <em>J Roy Stat Soc B</em>, Vol.68, pp.127-133.
  <br>
</LI>
<br>
</ul>

<!-- <h4>Journal Publications [Applications]</h4>
<ul> -->
	




<!-- </ul> -->


<h4>Invited Discussions and Reviews</h4>
<ul>
	
<li>
Gao, C. and Ma, Z. (2021)<br>
<a
href = "https://arxiv.org/abs/1811.06055">
Minimax rates in network analysis: Graphon estimation, community detection and hypothesis testing.</a>
<br>
<em>Statist Sci</em>, Vol.36(1), pp.16-33.
<!-- <a href="./software/colar.zip">[Matlab Code]</a> -->
<br>
</li>
<br>


<li>
Gao, C. and Ma, Z. (2020)<br>
<a>
<!-- href = "https://arxiv.org/abs/1811.06055"> -->
Discussion of `Network cross-validation by edge sampling'.</a>
<br>
<em>Biometrika</em>, Vol.107(2), pp.281-284..
<!-- <a href="./software/colar.zip">[Matlab Code]</a> -->
<br>
</li>
<br>

	
<li>
Low, M. and Ma, Z. (2015)<br>
Discussion: Frequentist coverage of adaptive nonparametric Bayesian credible sets. 
<br>
<em>Ann Statist</em>, Vol.43(4), pp.1448-1454.
<!-- <a href="./software/colar.zip">[Matlab Code]</a> -->
<br>
</li>
<br>
	



</ul>

<!----------------------------------------------------------------------->

</td>
</tr>




</table>

<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-8429489-2']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>



</body>

</html>
